iib/workers/tasks/build.py

```diff
@@ -337,6 +340,151 @@ def _get_resolved_image(pull_spec):
     return pull_spec_resolved
 
 
+def _get_index_database(from_index, base_dir):
+    """
+    Get database file from the specified index image and save it locally.
+
+    :param str from_index: index image to get database file from
+    :param str base_dir: base directory to which the database file should be saved
+    :return: path to the copied database file.
+    :rtype: str
+    :raises IIBError: if any podman command fails
+    """
+    cidfile = os.path.join(base_dir, 'cidfile.txt')
+    run_cmd(
+        ['podman', 'create', '--cidfile', cidfile, from_index],
+        exc_msg='Failed to create index image container',
+    )
+    with open(cidfile, 'r') as f:
```

Comment:
Hmm, this seems cleaner since podman might return other junk.

Reply:
`➜  iib-web git:(greenwave_conf) podman create 11866d6bc6aa ls
a4289a7fcba0be5515c8c3ed90c686f7ce012d4200145377e943225148477f99`

It actually only returns the container id. Am I missing anything?

Reply:
Well, I don't really know if this command can return something other than the container ID, so to eliminate this uncertainty I decided to use the cidfile since it was designed for this purpose.

In any case, I feel like it should be ok one way or the other.